content/release-10-4-6/overview-10-4-6.md

```diff
@@ -7,6 +7,8 @@ layout: bundle
 
 These release notes for **Cumulocity IoT release 10.4.6**, include all relevant changes (improvements and fixes) that have been implemented in Cumulocity IoT Core and integrated products since the previous GA release.
 
+You can find the corresponding product documentation for release 10.4.6 at [cumulocity.com/guides/10.4.6/about-doc/intro-documentation](https://cumulocity.com/guides/10.4.6/about-doc/intro-documentation/).

```

Comment:
I would actually not go further back then 10.7.0. We should not point people to the fact that all old documentation is still in the cloud :-).

Reply:
Done